### DISCUSSION TITLE
FleetDM Config File Location

### DISCUSSION THREAD

**Discussion by jblackley** [Security-Onion-Solutions/securityonion#10548] [Unsupported Versions] [answered]
I just turned up Security Onion 2.3.250 with FleetDM 4.31.1. I want to know where to find the FleetDM config file.

This stems from wanting to enable FleetDM Users and Software discovery. Documentation mentioned changing YAML files, but I didn't see anything telling me which files. I did find GitHub discussion #7844. @defensivedepth's answer got what I needed. Users and Software are both working.

However, fleet.config appears to be a temporary file. I can create the file, do an "ls", and see it. Later I look in the same directory and it's gone. I assume either "fleetctl apply" or "so-fleet-restart" is deleting it.

I may want to make more changes later. I assume there is some persistent config file where I can do this. In the docker context, I found /packs/osquery-config.conf, but it doesn't reflect the active config.

So...
1. Can someone tell me where the persistent config file is?
2. Or is FleetDM config a combination of files and defensivedepth's method is the easiest way of making changes?

**Comment by defensivedepth** [comment, marked as answer]
What kind of config options are you looking for?

There are FleetDM App config options & osquery endpoint config options.

**Reply by jblackley** [reply]
I don't know. The immediate need was to enable Users and Software, and your answer on discussion 7844 got it working. Going forward, I just want to make sure I understand how config works. Then I can do what I need without spending too much time or bothering anyone. But I don't have a specific need right now.
